shared/src/main/kotlin/com/egm/stellio/shared/util/ApiUtils.kt

```diff
@@ -168,16 +168,16 @@ internal fun canExpandJsonLdKeyFromCore(contexts: List<String>): Boolean {
     return expandedType == NGSILD_DATASET_ID_PROPERTY
 }
 
-enum class OptionsParamValue(val value: String) {
+enum class QueryParamValue(val value: String) {
```

Comment:
`TEMPORAL_VALUES` and `AGGREGATED_VALUES` should be moved / merged in `FormatValue`. `AUDIT` should be moved / merged in `OptionsValue`